packages/components/src/ui/data-table-filter/components/filter-selector.tsx

```
@@ -0,0 +1,261 @@
+  }, [property]);
+
+  useEffect(() => {
+    if (!open) setTimeout(() => setValue(''), 150);
+  }, [open]);
+
+  // biome-ignore lint/correctness/useExhaustiveDependencies: need filters to be updated
+  const content = useMemo(
+    () =>
+      property && column ? (
+        <FilterValueController
+          filter={filter!}
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)